source/includes/_authenticated_api_signatures.md

```diff
@@ -65,6 +65,92 @@ Find a member by email address. Once you have obtained a member id you can use t
 }
 ```
 
+### Create
+
+Creates a new signature for the specified petition. This API endpoint can be used to record externally recorded signatures for
+a particular petitions within your instance of the ControlShift platform. 
+
+All of the same post-signature actions that the platform usually supports including webhooks, CRM syncs, and other integrations
+will still be triggered, in the same way they are for normal signatures.
+
+
+| Field                     |   Explanation                                                              |
+| ------------------------- | ------------------------------------------------------------------------ |
+| `signature[first_name]`   | The first name of the member who took action                     |
+| `signature[last_name]`    | The last name of the member who took action                     |
+| `signature[email]`        | The email address of the member who took action                  |
+| `signature[postcode]`     | The postal or zip code of the member. Depending on your platform settings this may be a required field. Depending on your configuration, this may be validated for your country. |
+| `signature[phone_number]` | The phone number of the member who took action. Depending on your platform settings this may be a required field. |
+| `signature[locale]`       | The language and localization setting of the member who has taken action. This is specified as an [ISO_639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) two-character language code combined with an optional [ISO 3166](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) two character country code. For eg `en-US` or `en`.  |
```

Comment:
What happens if someone sends us an extended locale code here, like `de-DE`, that we don't explicitly support? Will we record it as `de` or ?

Reply:
It will return an error message.